doc/quick_start_guide/ffd/ffd.rst

```diff
@@ -20,14 +20,15 @@ Example designs
 ***************
 
 Demonstration
-===============================
+=============
 
-This is the far-field voice local command (FFD) example design with Sensory TrulyHandsfree™ (THF) speech recognition and local dictionary.
+This is the far-field voice local command (FFD) example design. Two examples are provided: both examples include speech recognition and a local dictionary. One example uses the Sensory TrulyHandsfree™ (THF) libraries, and the other one uses the Cyberon DSPotter™ libraries.
 
 When a wakeword phrase is detected followed by a command phrase, the application will output an audio response and a discrete message over |I2C| and UART.
 
+Sensory's THF and Cyberon's DSpotter™ libraries ship with an expiring development license. The Sensory one will suspend recognition after 11.4 hours or 107 recognition events, and the Cyberon one will suspend recognition after 100 recognition events. Ater the maximum number of recognitions is reached, a device reset is required to resume normal operation.To perform a reset, either power cycle the device or press the SW2 button.
 
-Sensory's THF software ships with an expiring development license. It will suspend recognition after 11.4 hours or 107 recognition events; after which, a device reset is required to resume normal operation. To perform a reset, either power cycle the device or press the SW2 button.
+Production software runs on a special device - need to contact Cyberon, Sensory or XMOS sale for information on that.
```

Comment:
This sentence needs work.  Perhaps, `Contact Cyberon, Sensory or XMOS sales for information about production use of the device.`  For my education, does the FFD example check for a key in the OTP?

Reply:
I used the suggestes sentence. As far as I know, only Cyberon needs a special key. @tomblackie-xmos, can confirm that.

Reply:
For production mode operation, the Cyberon library requires a specially programmed device, which has a distinct part number.  On other XMOS devices it will only operate for 100 recognions before stopping.  Sensory has a different mechanism.